slides/03-what-makes-a-model.qmd

```diff
@@ -259,23 +259,19 @@ ggplot(data = tree_preds, aes(age, latency)) +
 
 . . .
 
--   Workflows are smarter with new data than `model.matrix()` in terms of new factor levels
-
-. . .
-
--   You can use other preprocessors (more on feature engineering tomorrow!)
+-   Workflows handle new data better than `model.matrix()` in terms of new factor levels
 
 . . .
 
--   A workflow captures the entire modeling process
+-   You can use other preprocessors besides formulas (more on feature engineering tomorrow!)
 
 . . .
 
--   `fit()` and `predict()` apply to all the estimation steps
+-   They can help organize your work when working with multiple models
 
 . . .
 
--   They can help organize your work when working with multiple models
+-   [Most importantly]{.underline}, a workflow captures the entire modeling process. `fit()` and `predict()` apply to the preprocessing steps in addition to the actual model fit.
```

Comment:
I felt like it was worth putting this at the end and really hammering it home with some underlining. We could spend a second here talking about why it is important

It is also possible that this slide should come before the workflow diagrams?

Reply:
I guess we could move it first, but I tend to think "show" visually first then "tell" with words is a better order.

Reply:
That's fine! Order is probably less important than the fact that we should just try and mention that "the workflow captures the entire process" multiple times